imports/plugins/included/payments-stripe/server/methods/stripeapi.js

```diff
@@ -46,14 +46,12 @@ StripeApi.methods.getApiKey = new ValidatedMethod({
   name: "StripeApi.methods.getApiKey",
   validate: null,
   run() {
-    const settings = Packages.findOne({
-      name: "reaction-stripe",
-      shopId: Reaction.getShopId()
-    }).settings;
-    if (!settings.api_key) {
+    const settings = Reaction.getPackageSettings("reaction-stripe").settings;
+    const stripeConnectSettings = Reaction.getPackageSettings("reaction-stripe-connect").settings;
+    if (!settings.api_key && !stripeConnectSettings.api_key) {
```

Comment:
We should merge these two plugins (stripe connect and stripe) or we should have them be separate, but relying on stripe-connect to exist for the stripe plugin to function properly is probably a bad pattern.